src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -117,7 +117,7 @@ private static WebHostSettings GetDefaultSettings()
             bool isLocal = string.IsNullOrEmpty(home);
             if (isLocal)
             {
-                settings.ScriptPath = Path.Combine(HostingEnvironment.ApplicationPhysicalPath, @"..\..\sample");
```

Comment:
This only affects local debugging. We don't want the samples path to be hardcoded, so moving it to an app setting.